### PR TITLE
Tooltip on touch

### PR DESCRIPTION
**Pull request by vxsx**
tooltip wasn't consistently showing up in the content mode + additional fixes to make behaviour smoother

### CODE REVIEW

File: cms/static/cms/js/modules/cms.plugins.js [hunk 1]
```diff
@@ -277,11 +279,13 @@ var CMS = window.CMS || {};
...
-                this.ui.container.on(this.pointerOverAndOut, function (e) {
-                    e.stopPropagation();
+                this.ui.container.on(this.pointerOverAndOut + ' ' + this.touchStart, function (e) {
+                    if (e.type !== 'touchstart') {
+                        e.stopPropagation();
+                    }
                     var name = that.options.plugin_name;
                     var id = that.options.plugin_id;
-                    CMS.API.Tooltip.displayToggle(e.type === 'pointerover', e, name, id);
+                    CMS.API.Tooltip.displayToggle(e.type === 'pointerover' || e.type === 'touchstart', e, name, id);
```
Comment (FinalAngel) [inline cms/static/cms/js/modules/cms.plugins.js]: we should also pass an object there at some point

**Comment by FinalAngel** [comment]
looks like magic to mee, you probably need to run me through

File: cms/static/cms/js/modules/cms.plugins.js (hunk 1 above)
Comment (vxsx) [inline cms/static/cms/js/modules/cms.plugins.js]: yeah eventually

**Comment by FinalAngel** [comment]
:+1:

**Pull request merged**